Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
new file mode 100644
--- /dev/null	(date 1732829868284)
+++ b/.gitignore	(date 1732829868284)
@@ -0,0 +1,2 @@
+credentials/real-estate-property-scraper-4d9f71a7ded4.json
+credentials/*.json
Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1732020505679)
+++ b/.idea/.gitignore	(date 1732020505679)
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
